style.css

```diff
@@ -123,41 +146,51 @@ body {
 .subtitle-text {
     padding: 0;
     margin: 0;
+    font-size: 20px;
     -webkit-filter: var(--shadow);
     filter: var(--shadow);
 }
 
 #scene-selection {
+    height: 400px;
+    overflow: scroll;
```

Comment:
Dette burde være `overflow: hidden scroll;` for å unngå scroll bar horisontalt

Reply:
Lurt! hidden scroll blir best.